api/payara-bom/pom.xml

```diff
@@ -835,10 +835,15 @@
                 <version>${jsp-impl.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.glassfish.mq</groupId>
+                <artifactId>imq</artifactId>
+                <version>${openmq.version}</version>
```

Comment:
I guess this property is more or less _6.1.1_, but there is no such artifact deployed to Central. While building this bom-pom might not require such dependency to be available, having it feels strange...

Reply:
We have a patched version available in [our public patched repo](https://nexus.dev.payara.fish/#browse/browse:payara-artifacts:org%2Fglassfish%2Fmq).

Our build instructions are essentially out of date with how OpenMQ now publishes things - they changed all the artefact names.

This PR is just to correct these misaligned versions to stop Jenkins continually screaming and inconsistently failing about missing poms.

I may look at realigning these artefacts with the new upstream names in a separate PR.

Reply:
Realigning would require figuring our what generated https://central.sonatype.com/artifact/org.glassfish.mq/imq

The _end result_ is the `imq.jar` file present in the mq distribution zip, but this doesn't seem to match any of the other maven module names so presumably some renaming goes on _somewhere_ as a search for this file after the build only has results under:
```
./mq/dist/mq/lib/imq.jar
./mq/binary/share/opt/classes/imq.jar
```